webpack.config.js

```diff
@@ -79,6 +79,23 @@ if (process.env.RUN_ENV === 'PRODUCTION') {
     if (process.env.ESBUILD) {
       config.optimization.minimizer[0] = new EsbuildPlugin();
     }
+
+    config.module.rules.forEach(rule => {
+      // Remove devWarning
+      if (rule.test.test('test.tsx')) {
+        rule.use = [
+          ...rule.use,
+          {
+            loader: 'string-replace-loader',
+            options: {
+              search: 'devWarning(',
+              replace: 'if (false) devWarning(',
```

Comment:
```suggestion
              replace: 'if (process.env.NODE_ENV !== 'production') devWarning(',
```

参考 https://github.com/BerkeleyTrue/warning/blob/3c44be700a0e37e972c1a7e87446ba007f39cac0/warning.js#L21，这样不需要单独给 PRODUCTION 模式加。